packages/models/src/dto/v2/meeting.rs

```diff
@@ -0,0 +1,49 @@
+use bdk::prelude::*;
+use validator::Validate;
+
+#[derive(Validate)]
+#[api_model(base = "/v2/deliberations/:deliberation-id/meeting/:discussion_id", database = skip)]
+pub struct MediaPlacementInfo {
```

Comment:
```suggestion
#[api_model(base = "/v2/deliberations/:deliberation-id/meeting/:discussion_id", database = skip)]
#[serde(rename_all = "PascalCase")]
pub struct MediaPlacementInfo {
```
이렇게 하면 하나하나 지정할 필요가 없습니다. https://serde.rs/container-attrs.html 참고해주세요.